### PR TITLE
RF | _FolderTest

### PR DESCRIPTION
**Pull request by Rom-T**
https://trello.com/c/Lu2GplOP/918-rf-foldertest-testcreatefolderthatstartswithunsafecharacter

### CODE REVIEW

**Review by ValeriiSarukhanov — APPROVED** [review]

File: src/test/java/_FolderTest.java [hunk 1]
```diff
@@ -56,7 +55,7 @@ private static void createFolder(WebDriver driver, String folderName) {
         ProjectUtils.Dashboard.Main.NewItem.click(driver);
         driver.findElement(NAME).sendKeys(folderName);
```
Comment (GitForProjects) [inline src/test/java/_FolderTest.java:56]: Please use TestUtils.clearAndSend

File: src/test/java/_FolderTest.java [hunk 2]
```diff
@@ -243,32 +231,26 @@ public void testCreateFolderThatStartsWithUnsafeCharacter() {
     @Test
     public void testCreateFolderWithUnsafeCharacter() {
 
-        final String expectedErrorMessage = "» ‘@’ is an unsafe character";
-        final String expectedError = "Error";
-
         ProjectUtils.Dashboard.Main.NewItem.click(getDriver());
 
         getDriver().findElement(NAME).sendKeys("TestFolder@Jenkins");
```
Comment (GitForProjects) [inline src/test/java/_FolderTest.java:236]: Please use TestUtils.clearAndSend

File: src/test/java/_FolderTest.java (hunk 1 above)
Comment (Rom-T) [reply src/test/java/_FolderTest.java:56]: @GitForProjects Why do I need to clear the field if it is already empty. I think it's a redundant operation.

File: src/test/java/_FolderTest.java (hunk 2 above)
Comment (Rom-T) [reply src/test/java/_FolderTest.java:236]: @GitForProjects Why do I need to clear the field if it is already empty. I think it's a redundant operation.

**Review by GitForProjects — APPROVED** [review]
looks good

**Pull request merged**